frontend/views/containers/dashboard/GroupMembersTooltipPending.vue

```diff
@@ -0,0 +1,34 @@
+<template lang="pug">
+  tooltip(direction='bottom-end')
+    span.button.is-icon-small(data-test='pendingTooltip')
+      i.icon-question-circle
+    template(slot='tooltip')
+      i18n(
+        tag='p'
+        :args='{ invitedBy: `${invitedBy} ${invitedBy === ourUsername ? `(${L("you")})` : ""}` }'
```

Comment:
Simply:

```
:args='{ username }'
```

This sentence will not translate into all language properly otherwise.

Alternatively, you can construct two full sentences:

1. `This member did not use their invite link to join the group yet. This link should be given to them by {invitedBy}`
2. `This member did not use their invite link to join the group yet. This link should be given to them by {invitedBy} (you)`

By using computed properties and the `L` function

Reply:
Using the L function? How? I came up with this...
```pug
      i18n(
        ref='msgYou'
        v-if='invitedBy === ourUsername'
        tag='p'
        :args='{ invitedBy }'
      ) This member did not use their invite link to join the group yet. This link should be given to them by {invitedBy} (you).
      i18n(
        v-else
        ref='msg'
        tag='p'
        :args='{ invitedBy }'
      ) This member did not use their invite link to join the group yet. This link should be given to them by {invitedBy}.
```

Reply:
That uses way too many lines, so in this case a computed property is better. See for example `introTitle` in Payments.vue.

Reply:
That was what I thought. doing it